tablecodec/tablecodec.go

```diff
@@ -367,11 +368,8 @@ func DecodeRowWithMapNew(b []byte, cols map[int64]*types.FieldType, loc *time.Lo
 		}
 		idx++
 	}
-	// for decodeToMap:
-	// - no need handle
-	// - no need get default value
```

Comment:
Why remove the comments

Reply:
The parameters below are changed.

I added a comment at `DecodeRowToDatum`:
```go
// Default value columns, generated columns and handle columns are unprocessed.
func DecodeRowToDatum(b []byte, cols map[int64]*types.FieldType, loc *time.Location)
```